package.json

```diff
@@ -10,7 +10,11 @@
     "node": "10.23.1",
```

Comment:
could you remove this block, also add some scripts (mainly `osd` and `build`), reference here https://github.com/opensearch-project/notifications/blob/develop/dashboards-notifications/package.json#L12-L20

Reply:
Sure, will do.